src/celeritas/phys/PhysicsTrackView.hh

```diff
@@ -590,14 +590,11 @@ CELER_FUNCTION real_type PhysicsTrackView::range_to_step(real_type range) const
 {
     CELER_ASSERT(range >= 0);
     const real_type rho = params_.scalars.scaling_min_range;
-    if (range < rho)
+    if (range < rho * real_type(1.0000001))
```

Comment:
Revert since this is done in #479 (with slightly larger fudge factor).

Reply:
Whoops, this is what happens when you try to split up one accidentally large branch across three PRs. Thanks!